README.md

```diff
@@ -2,22 +2,26 @@
 
 Policy decision point (PDP) for web playback. 
 
-### Generate Swagger API documentation
+/auth endpoint takes an AccessRequest and grants or denies access by checking a list of policies.
 
-Some of the code is commented with the [Declarative Comments Format](https://github.com/swaggo/swag#declarative-comments-format).
-Which generates the swagger documentation.
+### Getting started 
 
-1. Download [Swag](https://github.com/swaggo/swag) for go by using:
+Start application with an example policy file:
 ```bash:
```

Comment:
```suggestion
```shell
```

Reply:
Fun fact, `bash` is an alias for `shell` https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L6443

Reply:
I wish we could be consistent with the naming too, but I think we need to solve it automatically. It steals a bunch of time and distracts from the _true_ errors or desgin flaws in PRs.

This is one reason why I wish we had a mono-repo solution instead of so many different repos, we fix it once and then the problem is gone **\*waves hand slowly\*** _forever_